frontend/src/components/Common/FirestoreDataProvider.tsx

```diff
@@ -112,7 +122,37 @@ export default function FirestoreDataProvider({ children }: Props): JSX.Element
         process.env.NODE_ENV === 'test' && children
       }
       {adminEmails == null || members == null || approvedMembers == null ? (
-        <Loader active={true} size="massive" />
+        <div style={{ marginTop: '45%' }}>
+          <Loader style={{ fontSize: 12 }} active size="massive"></Loader>
+          {!isIDOLMember && (
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'column',
+                justifyContent: 'flex-start',
+                padding: '0 15%'
+              }}
```

Comment:
I think there is enough styling in this file now to justify a separate css instead of inline.